config/cmds/create.go

```diff
@@ -110,6 +111,7 @@ func newCmdCreate() *cobra.Command {
 	setCmd.Flags().Int64Var(&req.pgPort, "pg.port", int64(5432), "Postgres port number")
 	setCmd.Flags().StringVar(&req.pgUser, "pg.user", "", "Postgres database user")
 	setCmd.Flags().StringVar(&req.pgPassword, "pg.password", "", "Postgres user password")
+	setCmd.Flags().StringVar(&req.pgMasterKeyURL, "pg.master_key_url", "", "Master key url for kms")
```

Comment:
```pg.master-key-url```

Reply:
It's better that way as in this command other flags are written using `_`.